unitTests/ui/AnalyticsTest.ts

```diff
@@ -378,5 +379,17 @@ export function AnalyticsTest() {
         });
       });
     });
+    describe('utilities methods', () => {
+      it('can add/remove new actionCause to actionCauseList', () => {
```

Comment:
I would split the tests in 2, one for adding and one for removing

Reply:
Alright will do!